cbmc/proofs/Makefile.common

```diff
@@ -42,9 +42,16 @@ CFLAGS += $(CFLAGS2) $(INC) $(DEF) -std=gnu99
 %.goto : %.c
 	$(GOTO_CC) -o $@ $(CFLAGS) $<
 
+# Use the generic implementations of atomic operations.
```

Comment:
We do not want this generic stuff in the common makefile.  Abstracting _IotMqtt_NextPacketIdentifier remove the need for this hack for the atomic add.

Reply:
Removed it, done!